### PR TITLE
 Patch kuttl test to work with releases

### PR DESCRIPTION
**Pull request by fdehay**
Patch init script to work for releases

| Q               | A
| --------------- | ---
| Bug fix?        | [X]

### What's in this PR?
Fix the image choosing algo to work with release tags in the kuttl ScaleUpandDown tests

### Why?
Kuttl tests were failing when release were made

### CODE REVIEW

File: test/e2e/kuttl/ScaleUpAndDown/00-createCluster.yaml [hunk 1]
```diff
@@ -3,7 +3,12 @@ kind: TestStep
 commands:
   - script: |
       BASEDIR=../../../..
-      VERSION=$(awk -F\" '/Version =/ {print $2}' $BASEDIR/version/version.go)-$CIRCLE_BRANCH
+      if [ $CIRCLE_TAG ]
+        then
+          VERSION="$(awk -F\" '/Version =/ {print $2}' $BASEDIR/version/version.go)-release"
+        else
+          VERSION="$(awk -F\" '/Version =/ {print $2}' $BASEDIR/version/version.go)-$CIRCLE_BRANCH"
+      fi
```
Comment (cscetbon) [inline test/e2e/kuttl/ScaleUpAndDown/00-createCluster.yaml:11]: @fdehay I would rather do 
```
VERSION=${CIRCLE_TAG:-CIRCLE_BRANCH}
```
I don't think we need to check the file at any point as we already use the right version in the tag

**Review by cscetbon — CHANGES_REQUESTED** [review]

**Comment by fdehay** [comment]
CIRCLE_TAG is empty if you are not releasing / tagging.
And if we don't check the file we do not know the current version


Le dim. 14 mars 2021 à 04:38, Cyril Scetbon ***@***.***> a
écrit :

File: test/e2e/kuttl/ScaleUpAndDown/00-createCluster.yaml (hunk 1 above)
Comment (cscetbon) [inline test/e2e/kuttl/ScaleUpAndDown/00-createCluster.yaml:11]:
```suggestion
     VERSION=${CIRCLE_TAG:-CIRCLE_BRANCH}
```

**Comment by cscetbon** [comment]
@fdehay any news on that PR ? Is it still needed ?

**Review by cscetbon — APPROVED** [review]

**Pull request merged**